test/helpers/nfv_helper.rb

```diff
@@ -0,0 +1,27 @@
+def set_nfv_data
```

Comment:
I do not like this method - what it does is that it creates a few instance variables, which is clear side effect and it is hidden from the user of this method.

I suppose that splitting this method into 3 functions with no creation of instance variables but just returning proper values would be more clean solution. 

Or just return it from this function so it can be used as `@nfv, @vnf_data, @auth = set_nfv_data` which is only a little longer call and gives user of this function control over where are data stored without unseen side effects.